styles.css/main.css

```diff
@@ -105,40 +105,84 @@ nav ul li {
   display: block;
   position: relative;
 }
+.bio a {
+  display: block;
+  position: relative;
+}
 
-.paperclip {
+img.paperclip {
```

Comment:
You need a more specific selector so that the properties here override some other img related rules you have